googleapis/google-cloud-node
# migrate code from googleapis/nodejs-policy-troubleshooter

- chore: add initial files for launch
- chore: add initial files for launch
- chore: add initial files for launch
- chore: add initial files for launch
- feat!: initial stub of library
- fix: naming on repo for status checks and submit samples (#11)
- chore: release 1.0.0 (#1)
- docs: update docs given new API name
- build: update gapic-generator-typescript to v1.2.10. (#13)
- chore: migrate to owl bot (#14)
- chore(deps): update dependency sinon to v10 (#30)
- chore(deps): update dependency ts-loader to v9 (#34)
- fix(deps): require google-gax v2.12.0 (#38)
- chore: new owl bot post processor docker image (#40)
- fix: use require() to load JSON protos (#41)
- chore: update gapic-generator-typescript to v1.3.2 (#42)
- chore: release 1.0.1 (#39)
- chore(deps): update dependency sinon to v11 (#45)
- fix: GoogleAdsError missing using generator version after 1.3.0 (#46)
- chore: release 1.0.2 (#47)
- chore: remove all monolith Bazel deps chore: release gapic-generator-csharp v1.3.7 chore: release gapic-generator-go 0.20.5 chore: release gapic-generator-java 1.0.14 chore: release gapic-generator-php 1.0.1 chore: release gapic-generator-python 0.50.0... (#53)
- docs(release-level): release level ga should be indicated (#54)
- fix(deps): google-gax v2.17.0 with mTLS (#57)
- chore: release 1.0.3 (#58)
- fix(deps): google-gax v2.17.1 (#60)
- chore: release 1.0.4 (#61)
- fix: Updating WORKSPACE files to use the newest version of the Typescript generator. (#62)
- chore: release 1.0.5 (#63)
- chore(nodejs): update client ref docs link in metadata (#68)
- fix(deps): google-gax v2.24.1 (#70)
- chore: release 1.0.6 (#71)
- feat: turns on self-signed JWT feature flag (#72)
- chore: release 1.1.0 (#73)
- chore: disable renovate dependency dashboard (#1194) (#75)
- fix(build): switch primary branch to main (#76)
- chore: release 1.1.1 (#77)
- build(node): update deps used during postprocessing (#1243) (#84)
- chore: fix the wrong post processor image (#86)
- chore(deps): update dependency @types/node to v16 (#85)
- chore(cloud-rad): delete api-extractor config (#88)
- chore(deps): update dependency sinon to v12 (#87)
- docs(samples): add example tags to generated samples (#89)
- build: add generated samples to .eslintignore (#90)
- docs(node): support "stable"/"preview" release level (#1312) (#93)
- chore: add api_shortname and library_type to repo metadata (#92)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#95)
- test(nodejs): remove 15 add 16 (#1322) (#97)
- build: update copyright year to 2022 (#99)
- chore: update v2.12.0 gapic-generator-typescript (#101)
- chore(deps): update dependency sinon to v13 (#103)
- docs(nodejs): version support policy edits (#1346) (#105)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#106)
- build(node): update client library version in samples metadata (#1356) (#116)
- chore(deps): update dependency @types/mocha to v9 (#117)
- chore(deps): update dependency sinon to v14 (#120)
- fix(deps): update dependency @google-cloud/local-auth to v2 (#123)
- build!: update library to use Node 12 (#122)
- chore(main): release 2.0.0 (#124)
- chore(deps): update dependency jsdoc-fresh to v2 (#127)
- chore(deps): update dependency jsdoc-region-tag to v2 (#128)
- fix(docs): describe fallback rest option (#129)
- chore(main): release 2.0.1 (#130)
- chore(deps): update dependency linkinator to v4 (#134)
- fix: better support for fallback mode (#135)
- fix: change import long to require (#136)
- chore(main): release 2.0.2 (#137)
- fix: do not import the whole google-gax from proto JS (#1553) (#139)
- fix: allow passing gax instance to client constructor (#140)
- chore(main): release 2.0.3 (#141)
- fix: preserve default values in x-goog-request-params header (#142)
- chore(main): release 2.0.4 (#143)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
